frontend/src/components/transactions/__tests__/__snapshots__/TransactionDetails.test.tsx.snap

```diff
@@ -894,7 +894,7 @@ exports[`<TransactionDetails /> renders no deposit 1`] = `
             <div
               className="ml-auto card-cell-text align-self-center col-md-12 col-auto"
             >
-              February 01, 2019 at 12:00:00am
+              co00on.5amt5_ti05.5amt5_ti05_for0amt
```

Comment:
Yeah this looks strange...

Reply:
It's date format `common.date_time.date_time_format` instead of `MMMM DD, YYYY [at] h:mm:ssa` applied to february the first.
`co<minutes>on.<day-of-week><am-pm>t<day-of-week-locale>_ti<minute><day-of-week-locale>.<day-of-week><am-pm>t<day-of-week-locale>_ti<minute><day-of-week-locale>_for<minute><am-pm>t